octovis/package.xml

```diff
@@ -20,12 +20,25 @@
   </export>
       
   <build_depend>octomap</build_depend>
-  <build_depend>libqglviewer-qt4-dev</build_depend>
+
+  <!-- Qt5 -->
+  <build_depend>libqt5-core</build_depend>
+  <build_depend>libqt5-opengl-dev</build_depend>
+
+  <!-- Qt4 -->
+  <!-- <build_depend>libqglviewer-qt4-dev</build_depend>
```

Comment:
Is there no libqglviewer dependency for Qt5 necessary? I think it should also be present, otherwise one with supplied in the sources will be built and used from the packages, which could lead to all sorts of version mismatches.

Reply:
I'll upstream a rosdep key, then we can add it